libs/ledger-live-common/src/exchange/swap/completeExchange.ts

```diff
@@ -12,6 +12,7 @@ import { delay } from "../../promise";
 import { ExchangeTypes, createExchange, getExchangeErrorMessage } from "@ledgerhq/hw-app-exchange";
 import type { CompleteExchangeInputSwap, CompleteExchangeRequestEvent } from "../platform/types";
 import { getProviderConfig } from "./";
+import { PayloadSignatureComputedFormat } from "@ledgerhq/hw-app-exchange/lib/Exchange";
```

Comment:
It is cleaner if you import `PayloadSignatureComputedFormat` in line 12 (with other hw-app-exchange imports) and add its export in hw-app-exchange/index.ts file.

Reply:
Thank you. Will do!